src/models/App.js

```diff
@@ -40,6 +40,27 @@ export const schema = new Schema({
       type: 'String',
     },
   },
+
+  // Hipchat webhooks
+  webhooks: [Number],
```

Comment:
In the end I left this on the App model because Studio doesn't actually have a channel id property. I think a reasonable caveat for Hipchat is 1 bot, 1 channel. In fact, the Hipchat add on installation workflow allows for this definition up front.